corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -385,7 +385,7 @@ def as_file(self):
 
     def get_http_response(self):
         headers = {}
-        content_length = self.payload.get_content_length()
+        content_length = self.get_content_length()
         if content_length is not None:
             headers['Content-Length'] = content_length
         return stream_response(self.payload.as_file(), headers)
```

Comment:
Yeah I think you're right. And `self.as_file()` is now just `self.payload` (Actually it's defined twice but I think the second time is a mistake.)